scripts/export_sdk_types.py

```diff
@@ -174,10 +174,20 @@ def export_thing(pattern, name, src, dst):
     export_type("ConsensusProtocols", src, dst)
     export_var("Consensus", src, dst)
     export_func("initConsensusProtocols", src, dst)
+    export_func("(cp ConsensusProtocols) DeepCopy", src, dst)
+    export_func("(cp ConsensusProtocols) Merge", src, dst)
     export_type("Global", src, dst)
     export_var("Protocol", src, dst)
     # do _not_ export init(), since go-algorand sets bounds, SDK does not
 
+    # Custom Consensus Functions
+    src = "config/config.go"
+    dst = "protocol/config/config.go"
+    export_func("SaveConfigurableConsensus", src, dst)
+    export_func("PreloadConfigurableConsensusProtocols", src, dst)
+    export_func("LoadConfigurableConsensusProtocols", src, dst)
+    # do not export SetConfigurableConsensusProtocols(), since go-algorand sets bounds, SDK does not
```

Comment:
I wonder if we should include a no-op `checkSetAllocBounds` function in the SDK so that we can sync this and `init()`.

Something for consideration in another PR.

Reply:
Created https://github.com/algorand/go-algorand-sdk/issues/757